src/views/Home.vue

```diff
@@ -1,16 +1,249 @@
-<template>
-  <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png">
-  </div>
-</template>
-
-<script>
-// @ is an alias to /src
-
-export default {
-  name: 'Home',
-  components: {
-    
-  }
-}
-</script>
+<template>
+  <div>
+    <InventoryModal
+      :open="showInventory"
+      :inventory="inventory"
+      @close="handleCloseInventory"
+    />
+    <div class="nav-icons">
+      <img
+        class="nav-icons__icon"
+        src="/favicon.ico"
+        @click="openPage('vry')"
+      />
+      <img
+        class="nav-icons__icon"
+        src="/images/discord-logo.png"
+        @click="openPage('discord')"
+      />
+      <img
+        class="nav-icons__icon"
+        src="/images/github.svg"
+        @click="openPage('github')"
+      />
+    </div>
+    <div class="match">
+      <div class="match__team match__attacking-team">
+        <PlayerCard
+          :team="attackingTeam"
+          @openInventory="handleShowInventory"
+        />
+      </div>
+      <div class="match__divider">
+        <p class="match__vs">vs</p>
+        <p class="match__time" v-if="timeStamp">
+          {{ momentAt(new Date(timeStamp * 1000)) }}
+        </p>
+      </div>
+      <div class="match__team match__defending-team">
+        <PlayerCard
+          :team="defendingTeam"
+          @openInventory="handleShowInventory"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import PlayerCard from "@/components/PlayerCard.vue";
+
+import rawJson from "@/assets/json/match.json";
+
+import InventoryModal from "@/components/InventoryModal.vue";
+
+import { FAVOURITE_WEAPONS } from "@/utils/weapons";
+
+import moment from "moment";
+
+export default {
+  components: { PlayerCard, InventoryModal },
+  mounted() {
+    let connection = new WebSocket("ws://localhost:1100/");
+
+    connection.onmessage = this.onMessage;
+
+    connection.onerror = () => {
+      this.timeStamp = null;
+      this.map = null;
+    };
+
+    if (this.attackingTeam.length == 0 && this.defendingTeam.length == 0) {
+      this.onMessage({ data: JSON.stringify(rawJson) });
+    }
+  },
+  data() {
+    return {
+      attackingTeam: [],
+      defendingTeam: [],
+      timeStamp: new Date().getTime() / 1000,
+      showInventory: false,
+      inventory: {},
+      pages: {
+        vry: "https://zaykenyon.github.io/VALORANT-rank-yoinker/",
+        github: "https://github.com/zayKenyon/VALORANT-rank-yoinker",
+        discord: "https://discord.gg/HeTKed64Ka",
+      },
+    };
+  },
+  methods: {
+    openPage(type) {
+      window.open(this.pages[type], "_blank");
+    },
+    momentAt(date) {
+      return moment(date).fromNow();
+    },
+    handleShowInventory(player) {
+      this.showInventory = true;
+      this.inventory = player;
+    },
+    handleCloseInventory() {
+      this.showInventory = false;
+      this.inventory = {};
+    },
+    onMessage(event) {
+      const { Players, time, map = {} } = JSON.parse(event.data);

```

Comment:
There needs to be added type checking for new request structure from https://github.com/zayKenyon/VALORANT-rank-yoinker/pull/113. This pull request added more data that could be send to websocket. There is now `type` field in JSON payload that can have values: `heartbeat`, `version`, `chat` or `matchLoadout`.